index.ts

```diff
@@ -106,12 +100,22 @@ export const DocumentCollection = {
     }
     return await apiGetDocument(id);
   },
-  // TODO: list documents
-  // async page({ args, context }) {
-  // },
+  async page({ self, args }) {
+    const { q: query, ...rest } = args;
+    const mimeType = 'application/vnd.google-apps.document';
+    const queryStr = query ? `and ${query}` : "";
+    const q = `mimeType='${mimeType}' ${queryStr}`;
+    const res = await api("GET", "www.googleapis.com", "/drive/v3/files", { ...rest, q });
+    const json = await res.json();
+    return { items: json.files, next: self.page({ ...rest, pageToken: json.nextPageToken }) };
+  }
 };
 
 export const Document = {
+  gref: ({ obj }) => {
+    return root.documents.one({ id: obj?.id });
```

Comment:
```suggestion
    return root.documents.one({ id: obj!.id });
```
You want it to throw if `obj` is ever undefined, as opposed to returning an incorrect gref